game/game.cpp

```diff
@@ -153,11 +153,11 @@ Game::~Game() {
 
 void Game::match(){
     createAll();
-    size_t turnCpt = 0;
+    idCurrentPlayer = 0;
```

Comment:
Pk l'avoir renommé idCurrentPlayer ?

Reply:
La variable compte juste les tours, elle ne revient pas à 0

Reply:
Ce qu'on peut faire pour garder le nb de tour et aussi l'id du joueur qui joue: 
idCurrentPlayer = turnCpt % this->nbPlayers;